PyDMXControl/_Colors.py

```diff
@@ -52,7 +95,25 @@ def add(color1: List[int], color2: List[int], percent1: float = 1, percent2: flo
 
     @staticmethod
     def to_dict(colors: List[int]) -> Dict[str, int]:
-        """ Assumes RGBWA """
+        """Convert a color from list form to a dict.
+        
+        Assumes RGBWA.
+        
+        Parameters
+        ----------
+        colors: Color to convert to a dictionary.
+
+        Returns
+        -------
+        Dict[str, int]
+            Color as a dictionary.
+
+        Examples
+        --------
+
+        >>> PyDMXControl.Colors.to_dict([1,2,3,4,5]) 
```

Comment:
```suggestion
        >>> Colors.to_dict([1,2,3,4,5]) 
```
Remove package name.